cypress/integration/tests/controls/businessservices/create.test.ts

```diff
@@ -69,7 +69,7 @@ describe("Business service validations", () => {
     });
 
     it("Business service unique name validation", function () {
-        const businessServices = new BusinessServices(data.getStakeholderName());
+        const businessServices = new BusinessServices(data.getCompanyName());
         // Create business service
```

Comment:
I would like to go with Name, Description and Comment etc.
These are the fake values; we dont need to check if that is companyName or any random string.

Reply:
those are generic values. It doesn't mean to you, but it has essence. Like you can pass a name in email field, similarly, you cannot use a person name as fake jobfunction name. using description, company name etc is too specific. They don't sound generic at all. Hence, these changes.